src/gui/transferlistmodel.cpp

```diff
@@ -526,6 +532,10 @@ QVariant TransferListModel::internalValue(const BitTorrent::Torrent *torrent, co
         return torrent->nextAnnounce();
     case TR_PRIVATE:
         return (torrent->hasMetadata() ? torrent->isPrivate() : QVariant());
+    case TR_PERCENT_SELECTED:
+        if (!torrent->hasMetadata())
+            return 0;
+        return (torrent->wantedSize() * 100) / torrent->totalSize();
```

Comment:
Did you realize you were using integer instead of floating point? Try the following:
```suggestion
        return torrent->hasMetadata() ? (qreal(torrent->wantedSize()) * 100 / torrent->totalSize()) : 0;
```